modules/e2ee/Context.js

```diff
@@ -100,6 +103,34 @@ export class Context {
         this._sendCount = BigInt(0); // eslint-disable-line new-cap
     }
 
+    /**
+     * Sets the public or private key used to sign or verify frames.
+     * @param {CryptoKey} public or private CryptoKey object.
+     * @param {Object} signature options. Will be passed to sign/verify and need to specify byteLength of the signature.
+     *  Defaults to ECDSA with SHA-256 and a byteLength of 132.
+     */
+    setSignatureKey(key, options = {
+        name: 'ECDSA',
+        hash: { name: 'SHA-256' },
+        byteLength: 132 // Length of the signature.
+    }) {
+        this._signatureKey = key;
+        this._signatureOptions = options;
+    }
+
+    /**
+     * Decide whether we should sign a frame.
+     * @returns {boolean}
+     * @private
+     */
+    _shouldSignFrame() {
+        if (!this._signatureKey) {
```

Comment:
`return Boolean(this._signatureKey);`

Reply:
i want to make this more complicated in the future so would like to keep this.